public-api/src/main/java/org/pmiops/workbench/publicapi/DataBrowserController.java

```diff
@@ -254,7 +379,7 @@
 
         if (conceptName != null && domain_id != null) {
             conceptList = conceptDao.findConceptLikeNameAndDomainId(conceptName,domain_id,std_concept);
-        } else if(conceptName != null && domain_id == null){
+        } else if((conceptName != null && !conceptName.isEmpty()) && domain_id == null){
```

Comment:
Does this handle conceptName = '' ?

Reply:
Yes, in the case of empty concept name it finds all the concepts ordered by count value as in the else clause.